packages/api/src/models/cleaningTaskModel.js

```diff
@@ -17,6 +17,58 @@ import Model from './baseFormatModel.js';
 import taskModel from './taskModel.js';
 
 class CleaningTaskModel extends Model {
+  // TODO: remove stub and update controller and FE to accept volume and weight
+  $parseDatabaseJson(json) {
+    // Remember to call the super class's implementation.
+    json = super.$parseDatabaseJson(json);
+    if (json.weight && json.weight_unit) {
+      json.product_quantity = json.weight;
+      json.product_quantity_unit = json.weight_unit;
+      delete json.weight;
+      delete json.weight_unit;
+    } else if (json.volume && json.volume_unit) {
+      json.product_quantity = json.volume;
+      json.product_quantity_unit = json.volume_unit;
+      delete json.volume;
+      delete json.volume_unit;
+    }
```

Comment:
The API response should be unchanged currently because of this code. 

I expect we will work on this for Loics product tables for Soil V2 or refactored tasks whichever comes first.

Reply:
I'm all for those changes :) I just wanted to find out when this should be addressed.

What I wanted to say is, when I have volume, I don't expect "weight" or "weight_unit" to be included in the response, but the actual response is below. Also, I think `else` case should be handled as well.
```
"cleaning_task": {
    "task_id": 956,
    "product_id": 35,
    "cleaning_target": null,
    "agent_used": true,
    "water_usage": null,
    "water_usage_unit": "l",
    "weight": null,
    "weight_unit": null,
    "product_quantity": 2,
    "product_quantity_unit": "l"
}
```

Reply:
So I deleted those properties. The else case I added a handler for the "to the database" case. But "from the database" should not need an else case because of the database checks that guarantee the correct unit and quantity type. I added a comment in the else case spot for clarity!

Reply:
I left comments to simplify the code. (Sorry I think yo've notified twice for the same comments!)